src/Process/3dTilesProcessing.js

```diff
@@ -4,7 +4,7 @@ function requestNewTile(view, scheduler, geometryLayer, metadata, parent) {
         view,
         requester: parent,
         layer: geometryLayer,
-        priority: 10000,
+        priority: parent ? 1.0 / (parent.distance + 1) : 100,
```

Comment:
I'm a bit worried about how the priority will interact with other layers. With values this small, 3d-tiles tiles will always be downloaded after terrain tiles, regardless of the precision of the data.

This is part of a broader issue about how to homogenize priority values though, so you can go ahead and merge.

Reply:
Yep, on the other hand, when we mixed pointcloud with 3dTiles, the old value made itowns wait for *all* 3dtiles building to be downloaded before even downloading the first point ;-) I'll say this is better than before, if not ideal.

Reply:
IMHO priorities only make sense relative to other commands for the same layer and provider.

Currently the scheduler uses the priority as an absolute value for a given host which is wrong... I have plans to fix this, I just need some free time :)

Reply:
I agree. Good to know you have plans for improving priority management.